.github/review/mcp_trial.py

```
@@ -0,0 +1,332 @@
+    expected_body, expected = validate_events(events, meta)
+    for value in expected:
+        diff = git(repo, "--literal-pathspecs", "diff", "--no-ext-diff", "--no-textconv", "--no-renames", merge_base, meta["head"], "--", value["path"])
+        require(value["line"] in _diff_right_lines(diff), "inline finding is outside the PR diff")
+    verify_live(meta, token); review = verify_review(meta, token, expected_body, expected)
+    evidence = {key: meta[key] for key in ("repo", "pr", "base", "head", "run", "attempt")}; evidence.update(model=model, variant=variant, diff_bytes=len(full.encode()), delta_bytes=len(delta.encode()), delta_identical=delta == full, review=review, sessions=sessions)
+    (output / "trial.json").write_text(json.dumps(evidence, indent=2), encoding="utf-8"); (output / "trial.json").chmod(0o600)
+    print(json.dumps({"ok": True, "review": review.get("html_url", ""), "diff_bytes": len(full.encode())})); return 0
+
+
+if __name__ == "__main__":
+    try: raise SystemExit(main(sys.argv))
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)